cupy/fallback_mode/data_tranfer.py

```diff
@@ -0,0 +1,42 @@
+"""
+Data transfer methods
+"""
+
+import cupy as cp
+import numpy as np
+
+
+def vram2ram(args, kwargs):
+
+    cpu_args = []
+    cpu_kwargs = {}
+
+    for arg in args:
+        if isinstance(arg, cp.ndarray):
+            cpu_args.append(cp.asnumpy(arg))
+        else:
+            cpu_args.append(arg)
+
+    for arg_name, arg in kwargs.items():
+        if isinstance(arg, cp.ndarray):
+            cpu_kwargs[arg_name] = cp.asnumpy(arg)
+        else:
+            cpu_kwargs[arg_name] = arg
+
+    return tuple(cpu_args), cpu_kwargs
+
+
+def ram2vram(res):
+
+    if isinstance(res, (list, tuple)):
+        gpu_res = []
+        for r in res:
+            if isinstance(r, np.ndarray):
+                gpu_res.append(cp.array(r))
+            else:
+                gpu_res.append(r)
+    else:
+        if isinstance(res, np.ndarray):
+            return cp.array(res)
+        else:
+            return res
```

Comment:
Another minor nit: I usually prefer to just write 'return res' instead of putting that inside the 'else'. Don't worry about changing it.

Reply:
Ah, okay. Done.